python/pyspark/sql/functions.py

```diff
@@ -1637,6 +1637,27 @@ def explode(col):
     return Column(jc)
 
 
+@since(2.1)
+def posexplode(col):
```

Comment:
For this one, I thought the reason is `explode` is already registered. `posexplode` is a pair of that.